documentation/closed-contributions.md

```diff
@@ -51,7 +59,7 @@ git commit
 git push internal
 ```
 
-Open a pull request when you are ready to receive feedback from stakeholders.
+Open a pull request when you are ready to receive feedback from stakeholders. You'll see the pull request in the internal repository.
 
 After any iterative work, close the pull request. Since the closed repository is a mirror of the open one, we do not merge changes to it.
```

Comment:
I'm wondering if we should move this step until later. I think some GitHub configs automatically delete a branch when a repo is closed

Reply:
Should not be applicable: the public branch is effectively a second branch as far as automation is concerned.